### DISCUSSION TITLE
Resizing preview box

### DISCUSSION THREAD

**Discussion by Pedroman1** [Acly/krita-ai-diffusion#470] [Q&A]
Hi maybe this is just a Krita setting not related to the wonderful amazing plugin we have here but I can't for the life of me figure out how to resize the preview box that shows the image gen under AI Image generation
<img width="459" alt="question" src="https://github.com/Acly/krita-ai-diffusion/assets/89334006/ffcdc49d-6aae-43fb-b1fb-973534942d1d">

. Is this possible? no worries if not. super minimal issue that doesn't necessarily need a fix but as I am new to krita I thought I'd ask... this plugin is badass i love it so much I use for all my image gen now

**Comment by Acly** [comment]
Not possible at the moment. Usually the preview is bigger though compared to the rest of the UI, did you configure a large font size in Krita that makes the text bigger?

**Comment by olegsatov** [comment]
Actually you can do it by manually editing pykrita/ai_diffusion/ui/generation.py

```
class HistoryWidget(QListWidget):
    _model: Model
    _connections: list[QMetaObject.Connection]
    _last_job_params: JobParams | None = None

    item_activated = pyqtSignal(QListWidgetItem)

    _thumb_size = 96    <==== THIS NUMBER 

    _applied_icon = Image.load(theme.icon_path / "star.png")
    _list_css = f"""
        QListWidget {{ background-color: transparent; }}
        QListWidget::item:selected {{ border: 1px solid {theme.grey}; }}

```

☝️    _thumb_size = 96 

In v1.15 it's line 50. I've set mine to 215 - it will fit 2 thumbs in a row.

Not the best way to do it, but...

<img width="529" alt="Снимок экрана 2024-03-07 в 23 05 52" src="https://github.com/Acly/krita-ai-diffusion/assets/147343262/d70b555e-2b9a-4c8d-9ee6-68bf94d2ae14">

**Comment by Pedroman1** [comment]
Awesome thanks for the quick reply and yes I did change the text good eye

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by MoeWa77n** [comment]
My eyes arent the best anymore, i would also like to change the text size of promt input.
Which file did you edit to increase the text size, can you please name the file name. Thanks

**Reply by chooks007** [reply]
i dare say you would have found the file by now but if not, and for anyone else, it can be found at,,, 
C:\Users\{User}\AppData\Roaming\krita\pykrita\ai_diffusion\ui

BTW if you were able to find and modify the text input region, could you please post the modified code. thanks
